package.json

```diff
@@ -11,6 +11,9 @@
     "offline",
     "mvc"
   ],
+  "bin": {
+    "installInfo": "./bin/installInfo.js"
```

Comment:
I need to try out on Windows, as you rely on the hash-bang logic of the unix shell here. Eventually we need to change this to "node ./bin/installInfo.js" to work cross-platform.